controller.js

```
@@ -1,19 +1,22 @@
 "use strict";
 
-const { milliSecPerDay } = require("./variables");
+const { helpMessage } = require("./variables");
 const { episodeDate, converter, submitAlbum } = require("./lib");
 
 function controller(msg) {
-    if (msg.content.startsWith("&episode")) {
-        const reply = episodeDate(msg);
-        msg.reply(reply);
+    const command = msg.content.trim();
+    let reply;
+
+    if (command.startsWith("&episode")) {
+        reply = episodeDate(command.replace('&episode', '').trim());
```

I'm wondering what's best between these three formulation:
```
reply = episodeDate(command.replace('&episode', '').trim()); // The one used here
reply = episodeDate(command.replace('&episode ', '')); // Avoids using trim but maybe not easier to read.
reply = episodeDate(command.split(" ")[1]);  // Another way of doing it.
```

---

This is fair, I will probably use the third one as it supports strings likes 
`"&episode         dd/mm/yyyy"`. Note the spaces. I am not sure we want to return an error for a missplaced space.

---

Now that I think about it, the third one may return an error if we juste use "&episode". Maybe we should handle cases where the command is only "&episode" followed by n white spaces and "&episode [arg]" separatly.

---

It makes more sense to keep the trim() call, since it will have to be done in order to support cases like `&episode [add more spaces here] tomorrow`.

I can move it to the episodeDate function, but it will result in a line with only the trim() call.

---

Fair point, I didn't considered having more that 1 spaces. Looking at the other function, we make a lot of argument parsing. Maybe having a function that takes a command which has the format
`&command [arg]` with one or more whitespace before and after `[arg]` could help the whole project.